forward-message/assets/index.html

```diff
@@ -1,99 +1,113 @@
+                        Live
+                    </aside>
+                </nav>
+            </header>
+        </div>
+        <main>
+            <div class="content">
+                <h1>
+                    <img src="success.svg" />
+                    <div>
+                        <p>Welcome!</p>
+                        <p>Your live application with Twilio is ready to use!</p>
+                    </div>
+                </h1>
+                <section>
+                    <h2>Get started with your application</h2>
+                    <p>
+                        Now that your code is deployed, here are the last steps you need to do to finish your app.
+                    </p>
+                    <p>
+                        This app will return the <a href="https://www.twilio.com/docs/sms/twiml">TwiML</a> required to
+                        forward an incoming SMS message to the number that is set in the environment variables.
+                    </p>
+                    <ol class="steps">
+                        <li>Text any message to your Twilio phone number</li>
+                        <li>Receive the response message from Twilio</li>
+                    </ol>
+                </section>
+                <section>
+                    <!-- MOCK: gets filled in by a Maestro template -->
+                    <h2>Make the application your own</h2>
+                    <form>
+                        <label for="twilio-phone-number">Configured Twilio phone number</label>
+                        <input type="text" id="twilio-phone-number" class="phone-number" readonly=true value="+1-43x-xxx-xx31">
+                        <div style="align-self: start; display: inline-block;">
+                            <div style="display:flex; justify-content:center; padding:1rem 0;">
+                                <a target="_blank" href=""
+                                   style="
+                                           background-color: white;
```

Comment:
i think the border background and text need to swap from the mocks i've seen, its a blue background (color of current border) with the text being white

Reply:
I differ from the mock a little bit here; during our design discussion on February 5th, it seems like there was minor consensus towards making it a secondary button (blue on white). This change didn't appear in the mocks after that though -- I should check with Kruthi on whether this should be a primary or secondary button.

Reply:
I just confirmed with Kruthi; it is meant to be a white-on-blue secondary button, but the change isn't reflected in our mocks yet.